site/sfguides/src/optimizing-network-operations-with-cortex-ai-call-transcripts-and-tower-data-analysis/optimizing-network-operations-with-cortex-ai-call-transcripts-and-tower-data-analysis.md

```diff
@@ -0,0 +1,104 @@
+author: Swathi Jasti
+id: optimizing-network-operations-with-cortex-ai-call-transcripts-and-tower-data-analysis
+summary: Optimizing Network Operations with Cortex AI - Call Transcripts and Tower Data Analysis
+categories: Getting-Started, Streamlit, Data-Science-&-Ai, app-development, GenAI, Cortex, Cortex Analyst
+environments: web
+status: Published 
+feedback link: https://github.com/Snowflake-Labs/sfguides/issues
+tags: Getting Started, Data Apps, GenAI, Cortex, Cortex Analyst, Streamlit
+
+# Optimizing Network Operations with Cortex AI - Call Transcripts and Tower Data Analysis
```

Comment:
@sf-gh-sjasti Change the title to "Optimizing Network Operations with Snowflake Cortex AI"